subgraphs/uniswap-forks/src/price/price.ts

```diff
@@ -218,7 +218,7 @@ export function getTrackedVolumeUSD(
   ) {
     return [
       tokenAmount0.times(price0USD),
-      BIGDECIMAL_ZERO,
+      tokenAmount0.times(price0USD),
```

Comment:
Is change for addressing #657 as well or for a separate issue?

Reply:
I did not mean to change this in this PR. I went ahead and removed it.